pyproject.toml

```diff
@@ -24,7 +24,7 @@ classifiers = [
     "Programming Language :: Python :: 3.12",
 ]
 dependencies = [
-    "pmdsky-debug-py == 8.0.4",
+    "pmdsky-debug-py >= 10.0.10",
```

Comment:
I bumped pmdsky-debug-py in #506. Please don't use `>=` here, as pmdsky-debug doesn't use semver for the symbols itself and future changes can and will break symbols used by SkyTemple.

Reply:
Oh, I see. I'll change it.